### ISSUE TITLE
Distribute CodeFormatter as NuGet package?

### ISSUE THREAD

**Issue by ghuntley** [dotnet/codeformatter#201]
Howdy,

Any plans to ship CodeFormatter as a NuGet package so it can be installed via the command line:

    PM> Install-Package CodeFormatter

Example usages, integration with a git-precommit hook or as part of a FAKE build script.

The GitHub guys (@shiftkey) have rolled their own package which is perfect but it has been customized for their coding convention used in Octokit -> https://www.nuget.org/packages/Octokit.CodeFormatter/1.0.0-preview

**Comment by challe** [comment]
Any updates on this?

**Comment by ghuntley** [comment]
CodeFormatter was a one time thing by Microsoft to bring their closed source code up to spec. It isn't maintained by Microsoft and there's no assigned owner. Huge shame. 

Personally I have wasted so much energy over the years as a consultant, team-lead and open-source maintainer about code style. It's stupid. Languages should autoformat by default. Was going some golang this afternoon and one thing that hits you immediately is just how GOOD having `gofmt` existing by default and an entire ecosystem using it.

**Comment by csr-nick** [comment]
@ghuntley agree. This project is MIT licensed; is anyone up for forking it and fixing the outstanding issues?

**Comment by hanabi1224** [comment]
I recently made a fork of this repo and created a nuget package: [Dotnet.CodeFormatter.BuildTask.Fork/](https://www.nuget.org/packages/Dotnet.CodeFormatter.BuildTask.Fork/).  Hope it helps. @ghuntley @challe @csr-nick

**Comment by jmarolf** [comment]
We hope https://github.com/dotnet/roslyn/tree/master/src/Tools/dotnet-format will be the replacement for this repo someday

**Comment by sharwell** [comment]
🔗 Which is published here: https://www.nuget.org/packages/dotnet-format/
